packages/custom_lint/test/cli_process_test.dart

```diff
@@ -278,7 +278,8 @@ Analyzing...
       'dependency conflict',
       () async {
         // Create two packages with the same name but different paths
-        final workspace = await createSimpleWorkspace(['dep', 'dep']);
+        final workspace =
+            await createSimpleWorkspace(['dep', 'dep'], local: true);
```

Comment:
Add a trailling comma, then format :)

```suggestion
            await createSimpleWorkspace(['dep', 'dep'], local: true,);
```